ext/pcaprub_c/pcaprub.c

```diff
@@ -785,7 +786,9 @@ rbpcap_dump(VALUE self, VALUE caplen, VALUE pktlen, VALUE packet)
 
   Data_Get_Struct(self, rbpcap_t, rbp);
 
-  gettimeofday(&pcap_hdr.ts, NULL);
```

Comment:
Why does this break compilation on BSD? Which kind of complain does the compiler show?